playbooks/responses/crash-report.md

```diff
@@ -0,0 +1,9 @@
+Could you please attach a crash dump to help us get more information? You can collect them by adding the following snippet to your main process code, before app.whenReady:
```

Comment:
Would we mark something like this `blocked/need-info`?

Reply:
Ohh, yes, we would - adding that, thanks!